addToCart/index.html

```diff
@@ -0,0 +1,28 @@
+<!<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
in general we miss other meta tags, please keep it in mind for the next projects